### PR TITLE
compliance updated in prestashop 1.6

### PR DESCRIPTION
**Pull request by tsysamit**
compliance updated in prestashop 1.6

### CODE REVIEW

File: modules/securesubmitpayment/payment.tpl [hunk 1]
```diff
@@ -7,21 +7,24 @@
...
+			<div id="securesubmitIframeCardCvv" class="ss-frame-container"></div>
+			<p class="error-message" id="gps-cvv-error"></p>
+		</div>
 		<br />
-		<button type="submit" class="securesubmit-submit-button">{l s='Submit Payment' mod='securesubmitpayment'}</button>
+		<div class="form-group required ">
+			<div id="submit_button" class="ss-frame-container"></div>
+		</div>
+		<br/>
+		<!--<button type="submit" class="securesubmit-submit-button">{l s='Submit Payment' mod='securesubmitpayment'}</button>-->
```
Comment (mayur-vachchewar) [inline modules/securesubmitpayment/payment.tpl:28]: Please remove this commented line.

**Review by mayur-vachchewar — CHANGES_REQUESTED** [review]

File: modules/securesubmitpayment/assets/js/securesubmit.js [hunk 2]
```diff
@@ -1,71 +1,108 @@
...
-      .bind('click', secureSubmitFormHandler);
+  function secureSubmitResponseHandler(response) {
+    var $form = $("form.securesubmit-payment-form");
+    if (response.message) {
+      $('.securesubmit-payment-form')
+        .before('<ul class="error-message alert alert-danger"><li>' + response.message + '</li></ul>');
+        var submit_button = document.getElementById('submit_button');
+        submit_button.classList.remove("disable-button");
+    } else {
+      //alert ('[' + response.token_value + ']');
```
Comment (MSmedal) [inline modules/securesubmitpayment/assets/js/securesubmit.js:14]:
```suggestion
```

File: modules/securesubmitpayment/assets/js/securesubmit.js [hunk 3]
```diff
@@ -1,71 +1,108 @@
...
-      $form.unblock();
-    } else {
-      //alert ('[' + response.token_value + ']');
-      $form.append("<input type='hidden' class='securesubmitToken' name='securesubmitToken' value='" + response.token_value + "'/>");
-      $form.submit();
+    GP.errors = function() {
+      $(".error-message").remove();
     }
   }
+
```
Comment (MSmedal) [inline modules/securesubmitpayment/assets/js/securesubmit.js:107]:
```suggestion
```

**Review by MSmedal — APPROVED** [review]
Looks great! Just a couple of minor clean-up items.

**Comment by tsysamit** [comment]
@MSmedal I have updated the code as per your comments. Please check and approve

**Review by MSmedal — APPROVED** [review]

**Pull request merged**